specificatie/uris.yaml

```diff
@@ -0,0 +1,25 @@
+openapi: 3.0.3
```

Comment:
Wil je uiteindelijk deze components opnemen  in de openapi.yaml? Of wil je dat in de operapi.yaml deze headers worden gebruikt met verwijzing naar deze aparte yaml?

Reply:
Indien je dit in een aparte yaml wil opnemen dan ga ik er vanuit dat die yaml uiteindelijk in Haal-Centraal-common zal komen te staan.

Reply:
Het is de bedoeling dat de uris.yaml bestand als aparte bestand in de Haal Centraal Common repo wordt geplaatst en wordt verwezen in de openapi.yaml in de andere Haal Centraal repo's. Voor de review is het niet handig om samenhangende concepten via twee p.r.'s aan te bieden